app/src/main/java/org/mozilla/focus/utils/Browsers.java

```diff
@@ -71,8 +71,11 @@
     // installed then higher priority one will be stored here
     private ActivityInfo firefoxBrandedBrowser;
 
+    private Context context;
+
     public Browsers(Context context, @NonNull String url) {
         final PackageManager packageManager = context.getPackageManager();
+        this.context = context;
```

Comment:
Ah, I think your previous patch actually did that :)
https://github.com/mozilla-mobile/focus-android/pull/1045/files

FYI: You can use "git commit --amend" to update your existing commit and then (force) push the branch to your repository to update this PR.